locations/models.py

```diff
@@ -36,8 +36,7 @@ class LocationPage(Page):
     intro = models.CharField(max_length=250)
     body = RichTextField(blank=True)
     tags = ClusterTaggableManager(through=LocationPageTag, blank=True)
-    latitude = models.FloatField(blank=True)
```

Comment:
I would prefer to keep the lat and long fields in the database model, just change the admin/editor to split them when editing and join them when writing back to the model